scipy/spatial/transform/rotation.py

```diff
@@ -1,12 +1,55 @@
 from __future__ import division, print_function, absolute_import
 
+__all__ = ['Rotation']
+
 import numpy as np
+import scipy.linalg
 
 class Rotation(object):
     """Rotation in 3 dimensions.
 
     This class will include initializers from different representations, converters
     and some useful algorithms such as quaternion slerp and rotation estimation.
+
     """
-    def __init__(self):
-        pass
+    def __init__(self, quaternions):
+        """Initialize class from possible normalized quaternions.
+
+        Parameters
+        ----------
+        quaternions : (N, 4) numpy.array
+                      Each row is a quaternion stored in scalar-last
+                      (x, y, z, w) format.
+
+        """
+        # Each row should have 4 numbers
+        assert(quaternions.shape[1] == 4)
+
+        # L2 norm of each row
+        norms = scipy.linalg.norm(quaternions, axis=1)
+
+        # Divide each row by its norm.
+        # In case quat is [4 x 4], ensure norm is broadcasted along each column
+        normalised_quat = quaternions / norms[:, None]
+        self._quat = normalised_quat
+
+    @classmethod
+    def from_quaternion(cls, quat):
```

Comment:
I think it would make sense to simply delegate everything to `__init__`. I suggest write the detailed documentation in here and mention `from_quaternion` in `__init__`. What do you think?

Reply:
Sure, that also works.